src/main/java/racingcar/domain/Cars.java

```diff
@@ -1,64 +1,72 @@
 package racingcar.domain;
 
 import java.util.ArrayList;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Set;
-import racingcar.policy.RandomValueMovePolicy;
+import racingcar.utils.MovableCondition;
 
 public class Cars {
-    private static final int MIN_CAR_COUNT = 1;
-
     private final List<Car> values;
 
-    public Cars(List<String> names) {
-        validateCount(names.size());
-        validateUnique(names);
-
-        this.values = createCars(names);
+    public Cars(List<Car> inputs) {
+        validateNotEmpty(inputs);
+        this.values = new ArrayList<>(inputs);
     }
 
-    private void validateCount(int count) {
-        if (isInvalidCount(count)) {
-            throw new IllegalArgumentException("자동차 대수는 1이상이어야 합니다.");
+    private void validateNotEmpty(List<Car> inputs) {
+        if (inputs == null || inputs.isEmpty()) {
+            throw new IllegalArgumentException("자동차는 1대 이상이어야 합니다.");
         }
     }
 
-    private boolean isInvalidCount(int count) {
-        return count < MIN_CAR_COUNT;
-    }
-
-    private void validateUnique(List<String> names) {
-        Set<String> uniqueNames = new HashSet<>(names);
-
-        if (names.size() != uniqueNames.size()) {
-            throw new IllegalArgumentException("자동차 이름은 중복될 수 없습니다.");
-        }
+    public Cars(String inputNames) {
+        this(createCars(inputNames));
     }
 
-    private List<Car> createCars(List<String> names) {
+    private static List<Car> createCars(String inputNames) {
         List<Car> cars = new ArrayList<>();
 
-        for (String name : names) {
+        for (String name : split(inputNames)) {
             cars.add(new Car(name));
         }
 
         return cars;
     }
 
-    public void moveAll(RandomValueMovePolicy movePolicy) {
+    private static String[] split(String input) {
+        return input.split(",");
+    }
+
+    public void moveAll(MovableCondition condition) {
         for (Car car : values) {
-            car.move(movePolicy);
+            car.move(condition);
         }
     }
 
-    public List<CarSnapshot> toSnapshots() {
-        List<CarSnapshot> snapshots = new ArrayList<>();
+    public List<CarStatus> statuses() {
```

Comment:
```suggestion
    public RaceResult statuses() {
```
일급 콜렉션이라 이 객체로 반환해도 되지 않을까?